custom-recipes/neo4j-export-nodes/recipe.json

```
@@ -148,6 +148,27 @@
             "description": "Nodes with this label will be deleted before run",
             "defaultValue": false,
             "visibilityCondition": "model.expert_mode"
+        },
+        {
+            "label": "Handling NULL Values",
+            "type": "SEPARATOR",
+            "visibilityCondition": "model.expert_mode"
```

Actually we should add the link to pandas in a description field of the separator because links in a separator are recognised as link (and not just string).
```suggestion
            "type": "SEPARATOR",
            "description": "More detailed documentation: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_table.html",
            "visibilityCondition": "model.expert_mode"
```